### PR TITLE
added logo

### PR DESCRIPTION
**Pull request by luisheres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,76 @@
+<img src="./static/crowsnest.png" alt="crowsnest logo" width="150">
+
 # crowsnest
 
-**TODO** - elevator pitch and logo
+A maritime monitoring solution.
```
Comment (freol35241) [inline README.md:5]: Too narrow in my opinion. What about use cases where we want to send commands?

File: README.md [hunk 2]
```diff
@@ -1,63 +1,76 @@
...
 
 ## Overview
 
 `crowsnest` is a microservice-based application with the main focus of being a reasearch platform for maritime operations.
```
Comment (freol35241) [inline README.md:9]: My bad...

```suggestion
`crowsnest` is a microservice-based application with the main focus of being a research platform for maritime operations.
```

File: README.md [hunk 3]
```diff
@@ -1,63 +1,76 @@
+<img src="./static/crowsnest.png" alt="crowsnest logo" width="150">
```
Comment (freol35241) [inline README.md:1]: 👍

File: README.md (hunk 2 above)
Comment (luisheres) [reply README.md]: updated.

**Review by freol35241 — APPROVED** [review]
🥳

**Pull request merged**